geopandas/tests/test_geodataframe.py

```diff
@@ -437,6 +437,16 @@ def test_to_crs_geo_column_name(self):
         assert utm.geometry.name == 'geom'
         assert all(df2.geometry.geom_almost_equals(utm.geometry, decimal=2))
 
+    def test_geodataframe_explode(self):
+        s = GeoSeries([MultiPoint([Point(1, 2), Point(2, 3)]), Point(5, 5)])
+        df = GeoDataFrame({'geometry': s, 'col': [1, 2]})
```

Comment:
let's call the geometry column 'geom', so we catch the case I commented for above (and then you need to pass `geometry='geom'` to make this clear to geopandas)